assets/scripts/app/__tests__/StreetEditable.test.js

```diff
@@ -29,7 +27,6 @@ jest.mock('../../streets/data_model', () => {
 describe('StreetEditable', () => {
   beforeEach(() => {
     jest.resetModules()
-    CSSTransition.mockImplementation(({ children }) => (children))
```

Comment:
you moved CSSTransition mock outside but this implementation differs from yours(the default, or the way it should be). 

The CSSTransition in StreetEditable did not have an `in` prop and therefore the children would not render. 
https://github.com/streetmix/streetmix/blob/master/assets/scripts/app/StreetEditable.jsx#L149
Not sure why this works in production but in the test you can see here: https://github.com/streetmix/streetmix/pull/1500/files#diff-96268f7f5dee7d2478852ef0643bd4bbL57
that the snapshot does not have the children anymore. 

This is probably a bigger issue than just changing the mock.